src/main/java/com/endava/soa_5/page_objects/HomePage.java

```diff
@@ -0,0 +1,87 @@
+package com.endava.soa_5.page_objects;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends Base {
+
+    private @FindBy(xpath = "//a[@class='ico-register']")
+    WebElement registerButton;
+    private @FindBy(xpath = "//a[@class='ico-login']")
+    WebElement logInButton;
+    private @FindBy(xpath = "//li[@id='topcartlink']/a[@class='ico-cart']")
+    WebElement shoppingCartButton;
+    private @FindBy(xpath = "//span[contains(text(),'Wishlist')]")
+    WebElement wishlistButton;
+    private @FindBy(xpath = "//input[@id='small-searchterms']")
+    WebElement searchInputField;
+    private @FindBy(xpath = "//input[@value='Search']")
+    WebElement searchButton;
+    private @FindBy(xpath = "//ul[@class='top-menu']//a[contains(text(), 'Books')]")
+    WebElement booksButton;
+    private @FindBy(xpath = "//ul[@class='top-menu']//a[contains(text(), 'Computers')]")
+    WebElement computersButton;
+    private @FindBy(xpath = "//ul[@class='top-menu']//a[contains(text(), 'Electronics')]")
+    WebElement electronicsButton;
+    private @FindBy(xpath = "//ul[@class='top-menu']//a[contains(text(), 'Apparel & Shoes')]")
+    WebElement apparelAndShoesButton;
+    private @FindBy(xpath = "//ul[@class='top-menu']//a[contains(text(), 'Digital downloads')]")
+    WebElement digitalDownloadsButton;
+    private @FindBy(xpath = "//ul[@class='top-menu']//a[contains(text(), 'Jewelry')]")
+    WebElement jewelryButton;
+    private @FindBy(xpath = "//ul[@class='top-menu']//a[contains(text(), 'Gift Cards')]")
+    WebElement giftCardsButton;
+
```

Comment:
please add **compare product list** button(link) from the bottom of the page please

Reply:
done